tests/unit-ext/module-empty.c

```diff
@@ -0,0 +1,70 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
```

Comment:
what about renaming `module-empty` => "test-ext-module-empty", so that all the unittest case under /unit-ext have similar name pattern